src/core/Spec.js

```diff
@@ -37,7 +40,8 @@ getJasmineRequireObj().Spec = function(j$) {
       failedExpectations: [],
       passedExpectations: [],
       deprecationWarnings: [],
-      pendingReason: ''
+      pendingReason: '',
+      duration: null,
```

Comment:
Please add this to the documentation comment above so the API docs will generate correctly.